http/middleware/log.go

```diff
@@ -15,7 +15,7 @@ type (
 	Logger interface {
 		// Log creates a log entry using a sequence of alternating keys
 		// and values.
-		Log(keyvals ...interface{})
+		Log(keyvals ...interface{}) error
```

Comment:
Do you understand why this change? seems we return nil everywhere?

Reply:
If we use goakit plugin, it will replace the stdlib Logger with gokit's Logger which implements the Logger interface like above. So, we can simply use the gokit logger along with this middleware instead of having a separate adapter for it.

Reply:
Right here - https://github.com/goadesign/plugins/pull/61/files#diff-9aaf94aa7f78e80b8d2568dbc56aef4aR103

Reply:
Ah yes thank you!